virtualizor.py

```diff
@@ -145,6 +144,13 @@ def wait_for_install_server(self, hypervisor, mac):
                     return lease['ipaddr']
             time.sleep(1)
 
+    def wait_for_gateway(self, hypervisor, mac):
```

Comment:
You should instead just rename `wait_for_install_server` to something more generic like `wait_for_dhcp_lease` and use it directly.